### PR TITLE
CODENVY-384: improve machine type labels

### PR DESCRIPTION
**Pull request by olexii4**
**_1 Upvote**_ Signed-off-by: Oleksii Orel oorel@codenvy.com
@ashumilova  @vitaliy-guliy 
selection_057

### CODE REVIEW

**Comment by ashumilova** [comment]
+1

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java [hunk 1]
```diff
@@ -124,27 +124,37 @@ public void execute() {
...
+*/
+                            final MachineDto machine = machinesByNameMap.get(recipe.getName());
+                            final String targetType;
+                            if (machine == null) {
+                                targetType = recipe.getType();
+                            } else {
+                                final MachineConfigDto machineConfig = machine.getConfig();
+                                targetType = machineConfig.isDev() ? machineLocale.devMachineCategory() : machineConfig.getType();
+                            }
+                            final Target target = new Target(recipe.getName(), targetType, recipe);
```
Comment (vitaliy-guliy) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:153]: Why do you use variables as final?

**Comment by vitaliy-guliy** [comment]
LGTM

File: plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java (hunk 1 above)
Comment (olexii4) [inline plugins/plugin-machine/che-plugin-machine-ext-client/src/main/java/org/eclipse/che/ide/extension/machine/client/targets/TargetsPresenter.java:153]: I want to be sure that it won't change)

**Comment by olexii4** [comment]
selection_058

**Pull request merged**